project3/style.css

```diff
@@ -0,0 +1,2586 @@
+    }
+
+    .category {
+        display: block;
+        padding: 4px;
+        padding-top: 16px;
+    }
+
+    input[type="radio"]:hover,
+    input[type="checkbox"]:hover {
+        border-color: var(--color-green);
+    }
+
+    input[type="radio"]:checked,
+    input[type="checkbox"]:checked {
+        background-color: var(--color-green);
+    }
+
+    input[type="text"]:focus,
+    input[type="email"]:focus,
+    input[type="number"]:focus {
+        background-color: var(--color-offwhite);
+        border-bottom: 5px solid var(--color-green);
+    }
+
+    input[type="radio"]:checked+label,
+    input[type="checkbox"]:checked+label {
+        color: var(--color-green);
+        text-decoration: underline var(--color-green);
+    }
+
+    textarea[id="powers"] {
+        height: 200px;
+        width: 100%;
+        margin-top: 12px;
+        margin-bottom: 30px;
+        border-color: 2px solid var(--color-grey);
+    }
+
+    textarea[id="describe"]:focus,
+    textarea[id="powers"]:focus {
+        border-bottom: 5px solid var(--color-green);
+        background-color: var(--color-offwhite);
+    }
+}
+
+/* -----------------------------------------------------------------------
+----------------------------------------------------------------------------- */
+
+@media(min-width: 920px) {
+    :root {
+        /*  ==================
+          COLOR PALETTE
+          ==================*/
+
+        --color-red: #c61a3a;
+        --color-green: #71f79f;
+        --color-offblack: #1e2019;
+        --color-grey: #d1d5de;
+        --color-offwhite: #fafafa;
+        --color-blue: #163346;
+        /* Reminder that you can name your variables anything you want, they just have to start with two dashes (--) */
+        /* ㅇㅣ거 하면 이름대로 색 바꿀 수 있음 */
+    }
+
+    *,
+    *::before,
+    *::after {
+        box-sizing: border-box;
+    }
+
+    .button-reset {
+        background: transparent;
+        border: 0;
+        cursor: pointer;
+        font-family: inherit;
+        font-size: inherit;
+        font-weight: inherit;
+        padding: 0;
+        text-align: inherit;
+    }
+
+    button[type="submit"] {
+        background-color: var(--color-green);
+        font-family: Teko;
+        font-weight: 600;
+        font-size: 120%;
+        text-transform: uppercase;
+        margin: auto;
+        padding-left: 20px;
+        padding-right: 20px;
+        padding-top: 22px;
+        padding-bottom: 18px;
+        margin-left: 312px;
+        margin-bottom: 100px;
+        margin-top: 40px;
+        border: 5px solid var(--color-green);
+    }
+
+    button:hover {
+        background-color: var(--color-offwhite);
+        border: 5px solid var(--color-green);
+    }
+
+    .content {
+        background-color: var(--color-offblack);
+        width: 900px;
+        height: auto;
+        margin: auto;
+        padding-left: 42px;
+        padding-right: 42px;
+        position: relative;
+    }
+
+    .box {
```

Comment:
This `.box` element's styles work out such that there is a huge empty space to the right of your form at 1440px wide; see this GIF to see the issue in action:
![2019-12-01 17 39 09](https://user-images.githubusercontent.com/2653980/69921632-b9cfad80-1461-11ea-93ed-47ec1ca33571.gif)

While I know this layout can be achieved without this blank element, even if you kept it, you would need to rewrite the media queries to avoid this horizontal scroll effect.